wherehows-web/app/components/dataset-aclaccess.ts

```diff
@@ -25,7 +25,9 @@ const millisecondDays = (day = 1) => day * 24 * 60 * 60 * 1000;
  * at least 1 day from now
  * @type {Date}
  */
-const minSelectableExpirationDate = new Date(Date.now() + millisecondDays());
+// Making min selectable date to be an hour from the current time allows the user to select the
+// next day on the calendar but normally not the current day unless it's 11PM
+const minSelectableExpirationDate = new Date(Date.now() + millisecondDays(1 / 24));
```

Comment:
I think the business logic is that the user can select an expiration data between 2 days from the current time and up to 7 days. This change makes its an hour from the current time, i.e. the same day. I don't believe that is correct except the business logic has changed? Did you confirm with @mars-lan before making these types of changes?

Reply:
@theseyi The user should be able to select 1 day timeframes, this change was requested by @mars-lan, I'm assuming based on a business logic change. The issue is with the method of `Date now() + 1 day` is when given to the library it doesn't let the user choose the "next day" maybe because the first selectable day is "rounded up" to the next day?

Reply:
I see, maybe use moment as @igbopie suggested to cover these edge cases.
As a user making the change at 8am, I should not be able to set it to expire that same day

Reply:
`moment().add(1, 'd').asMilliseconds()`? should suffice

Reply:
@theseyi ,
Okay, so the issue is still that the actual calendar library is still "rounding up" on the first selectable day so if you add 1 day with moment, you still are only able to select only the day after tomorrow as the first available day. Found a good solution though: `moment().endOf('day').valueOf()` sets the value to the very last part of "today" which rounded forward by the calendar library will start the first selectable date as tomorrow.

Reply:
Awesome!